AnkiDroid/src/main/java/com/ichi2/anki/DeckPicker.kt

```diff
@@ -2221,6 +2329,28 @@ open class DeckPicker :
         createDeckDialog.showDialog()
     }
 
+    /**
+     * Displays a dialog for creating a new deck.
+     *
+     * This function shows a `CreateDeckDialog` for the current `DeckPicker` context,
+     * with the title specified by the resource ID `R.string.new_deck` and the dialog type set to `DECK`.
+     * The dialog does not have a predefined parent deck, hence `null` is passed for the parent deck parameter.
+     *
+     * When a new deck is created through the dialog, the following actions are performed:
+     * 1. The deck list is updated by calling `updateDeckList()`.
+     * 2. The options menu is invalidated, which triggers a refresh of the menu items to reflect the changes.
+     *
+     * Finally, the dialog is displayed by calling `showDialog()` on the `CreateDeckDialog` instance.
```

Comment:
This feels over-documented - explain what the method does and let the developer read the implementation details if they're unambiguous

@Arthur-Milchior did you request this level of detail? You may want to improve on my suggestion, as it's a little bare

```suggestion
     * Displays a dialog for creating a new deck.
     *
     * @see CreateDeckDialog
```

Reply:
@david-allison I asked to reduce the quantity of documentation in https://github.com/ankidroid/Anki-Android/pull/16679#discussion_r1667599963.

However, I should note that, in a way, I did ask for too much documentation. This is because, during the first few weeks, Sanjay wrote very little documentation. Even for complex classes such as AnkiFragment. And since Sanjay was still not writing enough documentation, I told him I want to see too much documentation. That he doesn't hesitate to write a lot, we can always cut later. So, yeah, in a sense, I did ask to write that much documentation, as a first step. And later, I tried to explain why I wanted to keep part of the documentation and why we could remove other parts